delivery_carrier_label_postlogistics/views/stock.xml

```diff
@@ -17,6 +17,9 @@
         </group>
       </group>
     </field>
+    <xpath expr="//button[@name='action_generate_carrier_label']" position="replace">
+      <button name="action_generate_carrier_label" string="Create Shipping Label" type="object" attrs="{'invisible': ['|', '&amp;', ('picking_type_code', '!=', 'incoming'), ('state', '!=', 'done'), '&amp;', ('picking_type_code', '=', 'incoming'), ('state', 'not in',  ('assigned', 'done'))]}" />
+    </xpath>
```

Comment:
Prefer `position="attributes"` instead of `replace` to only change what is needed: `attrs` attribute.
This can also avoid future issues, see 2nd paragraph in https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#inherited-xml.

Reply:
fixed in https://github.com/grindtildeath/delivery-carrier/pull/4